project.yaml

```diff
@@ -742,6 +745,7 @@ Layer:
                 foot,
                 bicycle,
                 tracktype,
+                CASE WHEN surface IN ('unpaved', 'compacted', 'dirt', 'earth', 'fine_gravel', 'grass', 'grass_paver', 'gravel', 'ground', 'mud', 'pebblestone', 'salt', 'sand', 'woodchips', 'clay') THEN 'unpaved' ELSE 'paved' END AS surface,
```

Comment:
`int_surface`, for consistency - it goes with other `int_` prefixes e.g. `int_tunnel`, `int_tc_type`